README.md

```diff
@@ -45,23 +45,47 @@ docker run -p 6379:6379 -it --rm redislabs/redistimeseries
 
 You can also build and run RedisTimeSeries on your own machine.
 
+Major Linux distributions as well as macOS are supported.
+
 #### Requirements
--  build-essential
--  The RedisTimeSeries repository: `git clone https://github.com/RedisTimeSeries/RedisTimeSeries.git`
+
+First, clone the RedisTimeSeries repository from git:
+
+```
+git clone --recursive https://github.com/RedisTimeSeries/RedisTimeSeries.git
+```
+
+Then, to install required build artifacts, invoke the following:
+
+```
+cd RedisTimeSeries
+make setup
```

Comment:
@rafie please describe here what it actually installs and allow the user to do it manually (for example, I think build-essential on ubuntu is what we need)

Reply:
system-setup.py is pretty verbose as for what needs to be installed for each platform.
build-essentials is debian/ubuntu specific.